backend/tests/integration/records/serializers/data/cv_with_author_with_affiliations.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body>
+  <p><b>
+    <a href="https://localhost:5000/literature/637275237">
+      This is a title.
+    </a>
+  </b></p>
+      <p>John6 Doe (<a href="https://localhost:5000/institutions/637275238">Gent U.</a>)</p>
+          <br>
```

Comment:
in general `<p>` should have enough spacing why do we need `br` ?

Reply:
I wanted to have extra space between literature records. Should I remove it?

Reply:
oh right, in that case I do not know. it's up to @StellaCh